subsys/bluetooth/controller/ll_sw/nordic/hal/nrf5/radio/radio_nrf5_ppi_resources.h

```diff
@@ -0,0 +1,164 @@
+/*
+ * Copyright (c) 2021 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+#include <sys/util.h>
+#include "radio_nrf5_fem.h"
+
+#if defined(CONFIG_BT_CTLR_TIFS_HW) || !defined(CONFIG_BT_CTLR_SW_SWITCH_SINGLE_TIMER)
+
+/* PPI channel 20 is pre-programmed with the following fixed settings:
+ *   EEP: TIMER0->EVENTS_COMPARE[0]
+ *   TEP: RADIO->TASKS_TXEN
+ */
+#define HAL_RADIO_ENABLE_TX_ON_TICK_PPI 20
+/* PPI channel 21 is pre-programmed with the following fixed settings:
+ *   EEP: TIMER0->EVENTS_COMPARE[0]
+ *   TEP: RADIO->TASKS_RXEN
+ */
+#define HAL_RADIO_ENABLE_RX_ON_TICK_PPI 21
+
+/* PPI channel 26 is pre-programmed with the following fixed settings:
+ *   EEP: RADIO->EVENTS_ADDRESS
+ *   TEP: TIMER0->TASKS_CAPTURE[1]
+ */
+#define HAL_RADIO_RECV_TIMEOUT_CANCEL_PPI 26
+
+/* PPI channel 22 is pre-programmed with the following fixed settings:
+ *   EEP: TIMER0->EVENTS_COMPARE[1]
+ *   TEP: RADIO->TASKS_DISABLE
+ */
+#define HAL_RADIO_DISABLE_ON_HCTO_PPI 22
+
+/* PPI channel 27 is pre-programmed with the following fixed settings:
+ *   EEP: RADIO->EVENTS_END
+ *   TEP: TIMER0->TASKS_CAPTURE[2]
+ */
+#define HAL_RADIO_END_TIME_CAPTURE_PPI 27
+
+#else /* defined(CONFIG_BT_CTLR_TIFS_HW) || !defined(CONFIG_BT_CTLR_SW_SWITCH_SINGLE_TIMER) */
+
+#define HAL_RADIO_ENABLE_ON_TICK_PPI 6
```

Comment:
Please double verify this change compared to old code

```suggestion
#define HAL_RADIO_ENABLE_ON_TICK_PPI 2
```

Reply:
my bad. Checked other defines but did not find other error. Fixed.